flags.go

```diff
@@ -188,7 +313,48 @@ func parseFlagsJSON(r io.Reader) (map[string]Flag, time.Time, error) {
 		log.Print("[goforit] error parsing JSON file:\n", err)
 		return nil, time.Time{}, err
 	}
-	return flagsToMap(v.Flags), time.Unix(int64(v.UpdatedTime), 0), nil
+	// we first unmarshall into the intermediate FlagJSONFormat
```

Comment:
You can do some cool things with the `json.Unmarshaler` interface here. https://golang.org/pkg/encoding/json/#Unmarshaler . Then each type becomes responsible for unmarshaling itself, it makes the code easier to read usually.